### ISSUE TITLE
Small screen menu button only visually conveys state

### ISSUE THREAD

**Issue by scottaohara** [atomiks/30-seconds-of-css#28]
The menu button, which changes from a hamburger to an X on small screen view ports, does not appropriately convey state to assistive technologies. 

Expected:  

When the navigation is closed the button should have an `aria-expanded="false"`.
When the navigation is opened the button should have an `aria-expanded="true"`.

Doing so will convey the button as "collapsed" or "expanded", respectively, when a screen reader is running.

**Comment by atomiks** [comment]
Merged: #27

**Comment by scottaohara** [comment]
Hey @atomiks, this issue is separate from the pr you accepted. This issue hasn’t been taken care of.

Thanks

**Comment by atomiks** [comment]
Woops. I thought that was addressed there
